content/history/directors.md

```diff
@@ -17,3 +17,5 @@ It is <em>not</em> an official record - just an informational page.
 <script type="text/javascript">
   window.asf.directorsTimelineChart(document.getElementById('graph'), window.asf.getDirectorsTimelineData())
 </script>
+
+> This data comes from <a href='./data/directors.js'>`./data/directors.js`</a>
```

Comment:
Just out of curiosity .... is the ` intended here in order and the link is formatted differently then?

Reply:
Yes it's meant to display as a blockquote, works but that's way down on the page.